src/index.tsx

```diff
@@ -223,13 +236,19 @@ async function activatePlugin(
   // Commands
 
   commands.addCommand(CommandIDs.showNotebookJobs, {
-    execute: async args => showJobsPanel(args as IJobsModel),
+    execute: async args => {
+      if (args['launcher']) {
+        eventLogger('launcher.show-jobs');
+      }
```

Comment:
I'm a bit confused about the type of `args` here. We are casting it to `IJobsModel` below, and yet, the launcher command seems to only specify `args` to be `{ launcher: true }`. No action needed here, but this seems suspicious and a good place for bugs to hide.

Reply:
You should have access to the fork now.

Reply:
Done in https://github.com/3coins/jupyter-scheduler/pull/3